saleor/plugins/manager.py

```diff
@@ -157,18 +157,10 @@ def calculate_checkout_subtotal(
             )
             for line_info in lines
         ]
-        default_value = base_calculations.base_checkout_subtotal(
```

Comment:
Is `base_checkout_subtotal` needed after this change? I don't see that we use it anywhere else.

Reply:
No, I forgot to drop it.